api/src/main/java/javax/el/ImportHandler.java

```diff
@@ -167,7 +167,11 @@ public ClassLoader run() {
                 }
 
                 return Class.forName(className, false, classLoader);
-            } catch (ClassNotFoundException ex) {
+                // Some operating systems have case sensitive path names. An example is Windows if className is
+                // attempting to be resolved from a wildcard import a java.lang.NoClassDefFoundError may be thrown as
+                // the case for the type likely doesn't match. See https://bugs.java.com/bugdatabase/view_bug.do?bug_id=8024775
+                // and https://bugs.openjdk.java.net/browse/JDK-8133522.
+            } catch (ClassNotFoundException | NoClassDefFoundError ex) {
```

Comment:
+1  

Are you going to create an [upstream pull request](https://github.com/eclipse-ee4j/el-ri) also?  Let me know if you prefer me to do that.